src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionActions.tag

```diff
@@ -50,7 +50,7 @@
      data-toggle="tooltip"
      data-placement="top"
      style="display: inline-block; padding-right: 5px;">
-     <div class="btn-group margin-bottom-7px">
+    <div class="btn-group margin-bottom-7px">
```

Comment:
You mean that the wrong indentation was not introduced here, but in #7312 right? I'm okay with allowing this change here so that the indentation can be corrected.